apps/web/public/locales/en/translation.json

```diff
@@ -9,7 +9,7 @@
   "Fever or chills": "Fever or chills",
   "Go back": "Go back",
   "Headache": "Headache",
-  "Header Title": "COVID-19 Symptom Checker",
+  "AppName": "COVID-19 Self-Assessment Tool",
```

Comment:
probably need consistency to whether do "AppName" or "App Name"

Reply:
I'll do that in [HAPP-455](https://freshworks.atlassian.net/browse/HAPP-455)